src/Frontend/Components/AttributionColumn/PackageAutocomplete.tsx

```diff
@@ -51,6 +48,7 @@ type AutocompleteAttribute = Extract<
 interface Props {
   title: string;
   attribute: AutocompleteAttribute;
+  temporaryPackageInfo: DisplayPackageInfo;
```

Comment:
i don't think we gain much by calling this "temporary".
```suggestion
  packageInfo: DisplayPackageInfo;
```

Reply:
True.